examples for clickhouse exporter are broken

### Component(s)

exporter/clickhouse

### What happened?

## Description

There is an error in clickhouseexporter makefile go build command.

Also `tracegen` was removed from the repository. It needs to be replaced with `telemetrygen`.

## Steps to Reproduce
Run the following commands
```
cd exporter/clickhouseexporter
make local-run-example
```
## Expected Result

all containers are running.

## Actual Result

Got error:
```
cd ../../ && GOOS=linux go build -o ./local/otelcontribcol ./cmd/otelcontribcol
main module (github.com/open-telemetry/opentelemetry-collector-contrib) does not contain package github.com/open-telemetry/opentelemetry-collector-contrib/cmd/otelcontribcol
make: *** [Makefile:4: local-run-example] Error 1
```

After fixing the Makefile I still get the error
```
cd ../../ && GOOS=linux go build -o ../../local/otelcontribcol  -C cmd/otelcontribcol
cd example && docker-compose up -d
ERROR: build path /home/golov-rg/work/cloud/opentelemetry-collector-contrib/cmd/tracegen either does not exist, is not accessible, or is not a valid URL.
make: *** [Makefile:5: local-run-example] Error 1
```

### Collector version

2008304e57

### Environment information

## Environment
OS: Ubuntu 22.04
Compiler: go 1.20.9


### OpenTelemetry Collector configuration

```yaml
``
```


### Log output

_No response_

### Additional context

_No response_

--- a/exporter/clickhouseexporter/Makefile
+++ b/exporter/clickhouseexporter/Makefile
@@ -1,8 +1,8 @@
 include ../../Makefile.Common
 
 local-run-example:
-	cd ../../ && GOOS=linux go build -o ./local/otelcontribcol ./cmd/otelcontribcol
+	cd ../../cmd/otelcontribcol && GOOS=linux go build -o ../../local/otelcontribcol
 	cd example && docker-compose up -d
 recreate-otel-collector:
-	cd ../../ && GOOS=linux go build -o ./local/otelcontribcol ./cmd/otelcontribcol
+	cd ../../ && make otelcontribcol
 	cd example && docker-compose up --build otelcollector

--- a/exporter/clickhouseexporter/README.md
+++ b/exporter/clickhouseexporter/README.md
@@ -140,7 +140,7 @@ SELECT Timestamp as log_time,
        toString(Links.TraceId)
 FROM otel_traces
 WHERE ServiceName = 'clickhouse-exporter'
-  AND SpanAttributes['peer.service'] = 'tracegen-server'
+  AND SpanAttributes['peer.service'] = 'telemetrygen-server'
   AND Timestamp >= NOW() - INTERVAL 1 HOUR
 Limit 100;
 ```

--- a/exporter/clickhouseexporter/example/docker-compose.yml
+++ b/exporter/clickhouseexporter/example/docker-compose.yml
@@ -69,10 +69,11 @@ services:
     stop_signal: SIGKILL
 
   # Traces generator
-  tracegen:
+  telemetrygen:
     build:
-      context: ../../../cmd/tracegen/
+      context: ../../../cmd/telemetrygen/
     command:
+      - traces
       - --otlp-endpoint=otelcollector:4317
       - --otlp-insecure
       - --rate=100

